### DISCUSSION TITLE
classic custom writers deprecation

### DISCUSSION THREAD

**Discussion by lodwicz** [jgm/pandoc#10224] [General]
hello! Is there any deadline set to remove support for "classic" lua writers? I have debugged one to produce pdf output via plain tugboat pdftex, and I would not like to lose the work done on that. thanks.

**Comment by tarleb** [comment]
There's no timeline. I'd believe that classic writers will stay around *at the very least* for the entirety of the "pandoc 3" epoch, and the `pandoc.write_classic` function is going to stay available even longer.
